docs/source/getting_started/manufacturing/07_3d_visualization.ipynb

```
@@ -0,0 +1,379 @@
+    "import open3d as o3d\n",
+    "import tifffile\n",
+    "\n",
+    "def convert_tiff_xyz_to_pcd(tiff_path, pcd_path):\n",
+    "    # Load the TIFF as a NumPy array\n",
+    "    xyz_data = tifffile.imread(tiff_path)\n",
+    "    \n",
+    "    if xyz_data.ndim == 3 and xyz_data.shape[2] == 3:\n",
+    "        # If the data is in the shape (H, W, 3), reshape it to (H*W, 3)\n",
+    "        xyz_data = xyz_data.reshape(-1, 3)\n",
+    "    elif xyz_data.ndim == 2 and xyz_data.shape[1] == 3:\n",
+    "        # Already in Nx3 shape; no need to reshape\n",
```

Ruff (0.12.2)

Redefinition of unused `fo` from line 7

Remove definition: `fo`

(F811)